bot/exts/info/doc/_cog.py

```diff
@@ -6,12 +6,15 @@
 import textwrap
 from collections import defaultdict
 from contextlib import suppress
+from functools import partial
 from types import SimpleNamespace
 from typing import Dict, NamedTuple, Optional, Tuple, Union
 
 import aiohttp
 import discord
-from discord.ext import commands
+from discord import Colour, Embed, Message, NotFound, Reaction, User
```

Comment:
This messes with the `git blame` and could be considered unnecessary.

Reply:
Did it to shorten the `predicate_emoji_reaction` function signature.